core/iwasm/common/wasm_c_api.c

```diff
@@ -1902,10 +1902,21 @@ wasm_frame_func_offset(const wasm_frame_t *frame)
     return frame ? frame->func_offset : 0;
 }
 
+void
+wasm_frame_vec_clone_internal(Vector *src, Vector *out)
+{
+    bh_assert(src->num_elems != 0 && src->data);
+
+    bh_vector_init(out, src->num_elems, sizeof(WASMCApiFrame), false);
```

Comment:
Should check the return value, there is memory allocation in be_vector_init.

And seems we should `bh_vectory_destroy` it before init, it may have been initialized, and re-initializing it may cause memory leak?